### ISSUE TITLE
Crash on check for ListCharField

### ISSUE THREAD

**Issue by hologerry** [adamchainz/django-mysql#455]
Python Version: Python 3.6.5
Django Version: Django 2.0.4
Version: Django-MySQL 2.2.1

I use the `ListCharField`

When I execute `python manage.py runserver`, I encountered the following:

```
Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django/core/management/__init__.py", line 365, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django/core/management/base.py", line 332, in execute
    self.check()
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django/core/management/base.py", line 364, in check
    include_deployment_checks=include_deployment_checks,
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django/core/management/base.py", line 351, in _run_checks
    return checks.run_checks(**kwargs)
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django/core/checks/registry.py", line 73, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django/core/checks/model_checks.py", line 27, in check_all_models
    errors.extend(model.check(**kwargs))
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django/db/models/base.py", line 1200, in check
    errors.extend(cls._check_fields(**kwargs))
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django/db/models/base.py", line 1272, in _check_fields
    errors.extend(field.check(**kwargs))
  File "/home/gerry/anaconda3/envs/im2txt_demo/lib/python3.6/site-packages/django_mysql/models/fields/lists.py", line 184, in check
    if max_size > self.max_length:
TypeError: '>' not supported between instances of 'int' and 'NoneType'
```
Please help me, Thanks a lot.

**Comment by adamchainz** [comment]
You need to specify `max_length` for the field to work. I'll fix this crash though.

**Comment by adamchainz** [comment]
In fact, `max_length` is required by `CharField`, and that check is running, just this crash is preventing you from seeing that warning.

**Comment by hologerry** [comment]
Thanks 😄

**Comment by adamchainz** [comment]
Fix released in version 2.2.2 on PyPI
